### DISCUSSION TITLE
WANTED: Musician for TIC80 Golf

### DISCUSSION THREAD

**Discussion by Chris-Mingay** [nesbox/TIC-80#1498] [General]
Hi all, I'm looking for someone to make some TIC80 music for a game I'm working on, TIC80 Golf

![tic80golf](https://user-images.githubusercontent.com/71371350/126144300-2c37786a-f64d-48d7-985e-45f88878f4fd.png)

It will be a freeware game but I will pay you for your services.

If you're at all interested please can you show me some examples of music you've made in TIC80 and we can go from there.

I guess I am looking for:

- Title music
- Menu music
- Putting for eagle or better (short loop)
- Putting for birdie (short loop)
- Putting for par or worse (short loop)
- Hole in one fanfare (short)
- Birdie fanfare (short)
- Par fanfare (short)
- Bogey or worse fanfare (short)
- Round finished music

Thanks!

Chris

**Comment by borbware** [comment]
I'm interested! How many tracks would you need?

Here's a playlist of some TIC-80 tunes I've made: https://soundcloud.com/oggborbis/sets/tic-80-chiptunes

**Reply by Chris-Mingay** [reply]
Wow great tracks!

I guess I am looking for:

- Title music
- Menu music
- Putting for eagle or better (short loop)
- Putting for birdie (short loop)
- Putting for par or worse (short loop)
- Hole in one fanfare (short)
- Birdie fanfare (short)
- Par fanfare (short)
- Bogey or worse fanfare (short)
- Round finished music

does the above make sense?

I made a golf game in Pico8 and I copied some music from my favourite golf game, Everybody's Golf on the Vita, I honestly wouldn't mind if the title music was similar again!

Pico8 golf - https://chrismingay.itch.io/picogolf
Everybodys Golf music (which I copied for the title) - https://www.youtube.com/watch?v=L9OQ4lWudKk

Cheers!

**Reply by borbware** [reply]
Cool! So three full tracks, three short loops and four fanfares. 

When would you like to have the tracks? I'm on a vacation right now, so if you're in a hurry, you can try to find someone who can start working on music right away. In other words, I can't promise anything for July, but I know I could start working on the tracks next month at the latest!

PS. Everybody's Golf main menu theme has a nice vibe, I can use it as a reference for the title music :>

**Reply by Chris-Mingay** [reply]
Exactly that thank you! No rush at all, this is just a fun project. Before you spend any time working on this, can you let me know how much you think you'd want for the above mentioned tracks? (I see we're now following each other on Twitter, so can do via DM if preferred).

Thanks again and enjoy the rest of your vacation!

**Reply by borbware** [reply]
Thanks! And yeah DMs is fine!

**Comment by ChildishGiant** [comment]
I'm in no way musically talented, just here to say the game is looking great!

**Reply by Chris-Mingay** [reply]
Very kind thank you very much!
